.eslintignore

```diff
@@ -1,6 +1,7 @@
 node_modules
 dist
+lib
 fixtures
 coverage
 __snapshots__
-src/generated
+generated
```

Comment:
why this change? are there other generated folders?

Reply:
For some reason, eslint was linting the generated folders.  I couldn't figure out why, because the path `src/generated` seems correct.  I had assumed it was due to some win-platform thing.  I will retest and see if I can avoid this change or improve the path.

Reply:
I modified the path to be `**/src/generated` and it appears to work.  I can only surmise that `src/generated` was being interpretted as an absolute path from the root.  According to the docs and examples I found online, using ** for wildcard nested directories is necessary.